### ISSUE TITLE
Satchel of Helpful Goods sometimes is empty in random TBC instance

### ISSUE THREAD

**Issue by Maleficen** [dalaranwow/dalaran-wow#5564]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
While queueing in random instance for Burning Crusade, sometimes the reward (Satchel of Helpful Goods) comes empty and with no item. Already happened to me 4-5 times in different instances from Burning Crusade, but it never happened with classic instances.

**Expected behaviour**: Tell us what should happen instead
The reward should come at least with 1 rare quality item.

**Steps to reproduce the problem**:

1. Queue for a random Burning Crusade dungeon
2. Finish the instance and open the reward item

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_073118_211356](https://user-images.githubusercontent.com/42087862/43674403-4956fa7a-97dc-11e8-9721-4b831143da8a.jpg)
